blobit-services/src/main/resources/bin/setenv.sh

```diff
@@ -1,7 +1,7 @@
 # Basic Environment and Java variables
 
 #JAVA_HOME=
-JAVA_OPTS="-XX:+UseG1GC -Duser.language=en -Xmx4g -Xms4g -XX:+AggressiveOpts -XX:+UseFastAccessorMethods -Djava.net.preferIPv4Stack=true -XX:MaxDirectMemorySize=1g  -Djava.util.logging.config.file=conf/logging.properties"
+JAVA_OPTS="-XX:+UseG1GC -Duser.language=en -Xmx4g -Xms4g -Djava.net.preferIPv4Stack=true -XX:MaxDirectMemorySize=1g  -Djava.util.logging.config.file=conf/logging.properties"
```

Comment:
this is to make the service start on JDK12 "UseFastAccessorMethods" throws an error (no more supported)